source/particle/property/interface.cc

```diff
@@ -393,6 +393,40 @@ namespace aspect
         return (update & (update_default | update_values | update_gradients));
       }
 
+      template <int dim>
+      bool
+      Manager<dim>::plugin_name_exists(const std::string &name) const
+      {
+        return (std::find(plugin_names.begin(),plugin_names.end(),name) != plugin_names.end());
+      }
+
+      template <int dim>
+      bool
+      Manager<dim>::check_plugin_order(const std::string &first, const std::string &second) const
+      {
+        AssertThrow(plugin_name_exists(first),
+                    ExcMessage("Could not find a plugin with the name " + first + "."));
```

Comment:
It's often easier to read if you enclose information that's not part of the regular sentence in some sort of bracket:
```suggestion
                    ExcMessage("Could not find a plugin with the name <" + first + ">."));
```
Think of the case where the plugin name has multiple words with spaces in between.